qcfractal/interface/collections/dataset.py

```diff
@@ -155,9 +161,21 @@ def download(self, local_path: Optional[Union[str, Path]] = None, verify: bool =
             local_path = self._view_tempfile.name
 
         r = requests.get(self.data.view_url_hdf5, stream=True)
+
+        pbar = None
+        try:
+            if progress_bar:
+                file_length = int(r.headers.get('content-length'))
+
+                pbar = tqdm(total=file_length, initial=0, unit='B', unit_scale=True)
+        except Exception as e:
+            print(e)
```

Comment:
I would like to turn the exception into a warning here, but I'm not sure how.

Reply:
Why do you need the exception? You would just warn if you hit the execution right?